src/platform_impl/windows/window_state.rs

```diff
@@ -32,6 +32,7 @@ pub struct WindowState {
     pub modifiers_state: ModifiersState,
     pub fullscreen: Option<Fullscreen>,
     pub is_dark_mode: bool,
+    pub forced_theme: Option<Theme>,
```

Comment:
would it make sense to get rid of is_dark_mode and use only theme ?
probably a breaking change.

Reply:
So forcing a theme is added alongside the normal functionality of using the windows theme settings.
Also a window is not always use the preferred theme on old windows versions.

A approach could be extending the theme enum to be something like:
````rust
enum {
  OS_DETERMINED,
  FORCED_DARK,
  FORCED_LIGHT,
}
````
and `OS_DETERMINED` would be the default.


However this does not get rid of `is_dark` because using dark style can fail and it is used to check the OS preference.